### PR TITLE
Implements the edit command and open task in text editor

### PR DESCRIPTION
**Pull request by mmerkes**

### CODE REVIEW

File: pkg/storage/store.go [hunk 1]
```diff
@@ -149,6 +150,65 @@ func StartTask(id int) {
...
+	// Create a temporary file to allow the user to edit the task
+	_, err = f.Write([]byte(fmt.Sprintf("%s\n%s", task.Title, task.Description)))
+	handleError(err)
+
+	fpath := f.Name()
+
+	f.Close()
+
+	// Option the default editor on the OS
+	cmd := exec.Command(os.Getenv("EDITOR"), fpath)
```
Comment (MyannaHarris) [inline pkg/storage/store.go:183]: How does this know what file to edit?

Comment (mmerkes) [reply pkg/storage/store.go:183]: `fpath` is the path to the temporary file I created, which is passed in as an argument.

Comment (MyannaHarris) [reply pkg/storage/store.go:183]: Ah, so the file being closed doesn't affect the call to the editor, makes sense. Thanks

**Review by MyannaHarris — APPROVED** [review]

File: pkg/storage/store.go (hunk 1 above)
Comment (mmerkes) [reply pkg/storage/store.go:183]: No. I'm not sure if we need to close the file or not, but vim is taking over.

**Pull request merged**